src/System.Text.Primitives/System/Text/Parsing/PrimitiveParser.cs

```diff
@@ -43,5 +44,11 @@ public static partial class PrimitiveParser
             0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,             // 239
             0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF              // 255
         };
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static bool IsHexFormat(ParsedFormat format)
```

Comment:
Question: Why did we change this from a IsHexadecimal property on TextFormat to a helper method?